src/components/Header/Header.module.css

```
@@ -5,36 +5,74 @@
 \*------------------------------------*/
 
 /**
- * 1) 
+ * 1) Global container for the header component. This helps set the stage for header recipes across applications
  */
 .header {
+  display: flex;
   background-color: var(--eds-theme-color-primary-background);
   color: var(--eds-theme-color-body-foreground-inverted);
-  padding: var(--eds-size-2);
+  padding: var(--eds-size-1) var(--eds-size-2);
   position: relative;
+  width: 100%;
+  z-index: var(--eds-z-index-500);
 
   @media all and (min-width: $eds-bp-lg) {
-    padding: var(--eds-size-6) var(--eds-size-4) var(--eds-size-4)
-      var(--eds-size-4);
+    flex-direction: column;
+    width: auto;
+    padding: var(--eds-size-6) var(--eds-size-2-and-half) var(--eds-size-6)
+      var(--eds-size-2-and-half);
     height: 100vh;
   }
 }
 
-.header__inner {
-  display: flex;
-  align-items: center;
-  gap: var(--eds-size-2);
+/**
+ * Container around sticky header
+ * 1) This is added to
+ */
+.header-sticky-container {
+  height: 100vh;
+  transition: all var(--eds-anim-move-quick) var(--eds-anim-ease);
 
-  @media all and (min-width: $eds-bp-lg) {
-    flex-direction: column;
-    align-items: flex-start;
-    height: 100vh;
-    overflow: hidden;
+  &:hover,
+  &:focus-within {
+    width: 100%;
+  }
+
+  @media all and (max-width: $eds-bp-lg) {
+    height: auto;
+    position: static;
+    margin-bottom: var(--eds-size-4);
   }
 }
 
-.header__menu-button {
-  @media all and (min-width: $eds-bp-md) {
-    display: none;
+/**
+* Pinned sticky header
+* 1) Pins the sticky header to the top of the page. Transitions in from above the viewport
+*/
+.header--sticky.eds-is-pinned {
+  @media all and (max-width: $eds-bp-lg) {
+    position: fixed;
+    transform: translateY(0%);
+  }
+}
+
+/**
+* Sticky header with scrolled class applied
+* 1) Class is applied when the sticky header gets pinned/unpinned and animation is needed
+*/
+.header--sticky.eds-is-scrolled {
+  @media all and (max-width: $eds-bp-lg) {
+    transition: transform var(--eds-anim-fade-quick) var(--eds-anim-ease);
```

Can we get a [prefers-reduced-motion](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) check here?
```suggestion
    transition: transform var(--eds-anim-fade-quick) var(--eds-anim-ease);
    
    @media (prefers-reduced-motion) {
      transition: none;
    }
```